README.md

```diff
@@ -30,4 +30,12 @@ These are files that aren't run directly, but can be imported by your own script
     - `colcon test` run, to compare results across runs of tests.
 - There should be visualizations of these different aggregations.
 - All testing tools should output valid SARIF.
-    - Many Cobra files cannot be loaded due to invalid JSON, or non-conformance to the SARIF spec.
\ No newline at end of file
+    - Many Cobra files cannot be loaded due to invalid JSON, or non-conformance to the SARIF spec.
+
+## 4. Snap
+These tools are also available to install from the [Snap Store](https://snapcraft.io/about) as a [snap](https://snapcraft.io/process-sarif).
+### Install: 
+`snap install process-sarif`
+### Usage
+Navigate to a directory containing the SARIF file(s) you wish to analyze. Run any of the available commands (sarif-load-test, conformance, duplicates, visualize), as `process-sarif.<app_name>` (ie: process-sarif.sarif-load-test)
```

Comment:
As discussed in the meeting earlier, I would love some wording here to the effect of this being a non-default method of installation that is not maintained by core Space ROS contributors but is instead maintained by Canonical.

Reply:
@EzraBrooks Thanks for the review, and for the discussion. As agreed, I have adjusted the description in the README to make it clear that the snap is only a complementary packaging option maintained by Canonical for the time being. We will measure user adoption to decide what will happen to it in the future.